README.md

```diff
@@ -124,6 +124,7 @@
 * [flake-utils](https://github.com/numtide/flake-utils) - Pure Nix flake utility functions to help with writing flakes.
 * [flake-utils-plus](https://github.com/gytis-ivaskevicius/flake-utils-plus) - A lightweight Nix library flake for painless NixOS flake configuration.
 * [gitignore.nix](https://github.com/hercules-ci/gitignore.nix) - The most feature-complete and easy-to-use `.gitignore` integration.
+* [haumea](https://github.com/nix-community/haumea) - Filesystem-based module system for Nix.
```

Comment:
My first reaction was "how is this different from `std` or `digga`?" and while that question is answered in the `README` for the repository, it might also be good to "leverage" some of the language from that section here to make it a little clearer.

Something like "Filesystem-based library framework for Nix", possibly specifying "Nix flakes" or "Nix expressions" instead of just "Nix"...? There's probably better words to use, since I only skimmed the documentation and code for the repositories....

Reply:
Updated the description to be a more descriptive. I didn't use any of your suggestions because it is not specific to libraries or flakes, even though that was my motivation.